src/js/const/idle.const.js

```diff
@@ -0,0 +1,3 @@
+export const IDLE = Object.freeze({
+  time: 1000 * 60 * 15, // milliseconds
```

Comment:
All developers know that JS uses milliseconds for timings. But if you want to leave a useful comment, you can write what this time will be as a result.